espanol/comida/victoria/victoria.md

```diff
@@ -1,3 +1,4 @@
 El capitán tras presentarse anter el rey, y mostrar su respeto con una gran reverencia.
 
-Le comunicó que su ejército había ganado la batalla decisiva contra el país vecino.
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.